README.md

```diff
@@ -115,8 +115,8 @@ Other Style Guides
     var b = 2;
 
     // good
-    const a = 1;
-    const b = 2;
+    const A = 1;
```

Comment:
Capitalizing variable names not Required. camelCase is preferred for naming conventions.